src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -185,6 +185,21 @@ const MetaTraderUI = (() => {
                     login   : () => (`${info} (${/demo/.test(accounts_info[acc_type].account_type) ? localize('Demo Account') : localize('Real-Money Account')})`),
                 };
                 $(this).html(typeof mapping[key] === 'function' ? mapping[key]() : info);
+
+                if (key === 'login') {
+                    const mt_financial_company = State.getResponse('landing_company.mt_financial_company');
```

Comment:
better to do this outside of the loop, no need to fetch it every time, same as below

Reply:
Noted and addressed in [#8fa5c77](https://github.com/binary-com/binary-static/pull/5006/commits/8fa5c77fa22915f8134b94b78381cc1f9114fe13)